untitled.js

```diff
@@ -0,0 +1,14 @@
+var name = 'Christian'
+let age = 15
+const gender = 'male' 
+let grade = 9;
+let height = 5,9;
+let weight = 155;
+const bday = 3/12;
+let todayismonday = {false;}
```

Comment:
This isn't a valid object. You probably wanted for this to be a boolean. In that case, simply `false` would work.